scripts/linux/ubuntu-server-18.sh

```diff
@@ -78,10 +55,10 @@ sudo systemctl restart sshd
 
 ### Create the clean script. ###
 echo '> Creating the clean script ...'
-sudo cat <<EOF > /tmp/clean.sh
+sudo cat <<EOF > /home/$BUILD_USERNAME/clean.sh
```

Comment:
Moving to `/home/$BUILD_USERNAME/` in preparation for `/tmp` to have `fsoptions` of `noexec`